### DISCUSSION TITLE
Dual boot vanilla OS 2 (orchid) guide

### DISCUSSION THREAD

**Discussion by maxed4901** [Vanilla-OS/handbook#109] [Q&A]
I am trying to install vanilla OS 2 as dual boot.

The instructions for dual boot has been greyd out.

Currently in advanced installation menu there are
1. boot partition (1GB as per VOS 1)
2. EFI partition (1GB as per VOS1) can be between 100 to 300 mb?
3. Root partition (-----) size 
4. Var partition (-----) size
5. Swap partition (----) 2-4gb recommended 

The var partition is maybe the AB partition? 

Waiting for updated instructions. 

Thanks.

**Comment by maxed4901** [comment]
1. Boot Partition: 1 GB (ext4 format)
2. EFI Partition: 512 MB (fat32 format)
3. Root Partition: 20.5 GB (unformatted initially)
4. Swap Partition: Optional for hibernation (default is zram)
5. Var Partition: Use the remaining storage (btrfs format) 

(https://github.com/Vanilla-OS/vanilla-installer/issues/162) 

(https://github.com/Vanilla-OS/vanilla-installer/issues/275).

Can some one confirm if this is correct?

**Reply by kbdharun** [reply]
Nope, a proper dual boot isn't currently supported in Orchid. As a workaround, you can switch between the boot entries manually from UEFI Firmware settings to dual boot.
